### PR TITLE
Adding information about TCGA BED files

### PR DESCRIPTION
**Pull request by tkoganti**
### Purpose/implementation Section
Adding documentation requested [here](https://github.com/AlexsLemonade/OpenPBTA-manuscript/issues/71#issuecomment-612973950)

#### What scientific question is your analysis addressing?
TCGA documentation

#### What GitHub issue does your pull request address?
https://github.com/AlexsLemonade/OpenPBTA-manuscript/issues/71#issuecomment-612973950

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.
This file was updated - `data/data-files-description.md`

#### Which areas should receive a particularly close look?
`data/data-files-description.md`

### CODE REVIEW

File: doc/data-files-description.md [hunk 1]
```diff
@@ -81,13 +81,13 @@ This document contains information about all data files associated with this pro
...
+|`tcga_6k_genes.targetIntervals.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#snv-and-indel-calling) | hg19 WXS target capture regions downloaded from GDC website
+|`whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.Gh38.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#snv-and-indel-calling) | hg38 version of `whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.bed`. Generated using CrossMap and bedtools sort and merge. Script in [TCGA-capture-kit-analyses](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/scripts/CrossMap.sh)
+|`whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#snv-and-indel-calling) | hg19 WXS target capture regions downloaded from GDC website
```
Comment (cansavvy) [inline doc/data-files-description.md:89]: An exact link to where these files are available at each of these GDC website references would be helpful!

File: doc/data-files-description.md [hunk 2]
```diff
@@ -81,13 +81,13 @@ This document contains information about all data files associated with this pro
...
+|`intersected_whole_exome_agilent_plus_tcga_6k_AND_tcga_6k_genes.Gh38.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#snv-and-indel-calling) | Generated using bedtools intersect from `tcga_6k_genes.targetIntervals.Gh38.bed` and `whole_exome_agilent_plus_tcga_6k.targetIntervals.Gh38.bed`
+|`tcga_6k_genes.targetIntervals.Gh38.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#snv-and-indel-calling) | hg38 version of `tcga_6k_genes.targetIntervals.bed`.  Generated using CrossMap and bedtools sort and merge. Script in [TCGA-capture-kit-analyses](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/scripts/CrossMap.sh)
```
Comment (cansavvy) [inline doc/data-files-description.md:86]: This link to this script is very helpful! Thanks!

Comment (cansavvy) [reply doc/data-files-description.md:86]: If you have any more of these kinds of scripts you can link to: even better!

**Comment by tkoganti** [comment]
NGS checkmate doen for all samples - https://cavatica.sbgenomics.com/u/d3b-bixu/dev-canine-workflow/tasks/5142ec02-9435-4aa6-ab0d-713cbdd4bb52/

**Comment by tkoganti** [comment]
All files in data description have links on how they were generated and two files were generated based on simple bedtools intersect command and I mentioned that in the `File description` field

**Comment by jashapiro** [comment]
Documentation change only. Merging without further tests.

**Pull request merged**